File: 1580-shuffle-the-array/1580-shuffle-the-array.java
class Solution {
    public int[] shuffle(int[] nums, int n) {
        int idx = 0;
        int[] arr = new int[2 * n];
        for (int i = 0; i < n; ++i) {
            arr[idx++] = nums[i];
            arr[idx++] = nums[n + i];
        }
        return arr;
    }
}

File: 1580-shuffle-the-array/1580-shuffle-the-array.cpp
class Solution {
public:
    std::vector<int> shuffle(std::vector<int>& nums, int n) {
        std::vector<int> result;
        for (int i = 0; i < n; ++i) {
            result.push_back(nums[i]);
            result.push_back(nums[n + i]);
        }
        return result;
    }
};